pytorch_lightning/callbacks/swa.py

```diff
@@ -156,7 +156,9 @@ def on_train_epoch_start(self, trainer: 'pl.Trainer', pl_module: 'pl.LightningMo
             self._average_model = self._average_model.to(self._device or pl_module.device)
 
             optimizers = trainer.optimizers
-            lr_scheduler = trainer.lr_schedulers[0]["scheduler"]
+
+            for param_group in optimizers[0].param_groups:
```

Comment:
is this limited to 1 optimizer right now?

Reply:
Yes, it is.